tests/form.spec.ts

```diff
@@ -1,469 +1,520 @@
-var Promise = require('bluebird');
-var inquirer = require('inquirer');
-var visuals = require('resin-cli-visuals');
-var m = require('mochainon');
+import Promise from 'bluebird';
+import * as sinon from 'sinon';
+import visuals from 'resin-cli-visuals';
+import { expect, use as chaiUse } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+chaiUse(chaiAsPromised);
 var form = require('../build/form');
```

Comment:
Why keeping `var` here instead of const?

Reply:
Sure, I guess I only did so in the major